clang/lib/StaticAnalyzer/Checkers/MallocChecker.cpp

```diff
@@ -1122,7 +1157,7 @@ MallocChecker::performKernelMalloc(const CallEvent &Call, CheckerContext &C,
   if (TrueState && !FalseState) {
     SVal ZeroVal = C.getSValBuilder().makeZeroVal(Ctx.CharTy);
     return MallocMemAux(C, Call, Call.getArgExpr(0), ZeroVal, TrueState,
-                        AF_Malloc);
+                        AllocationFamily(AF_Malloc));
```

Comment:
Now I get what you meant when pushed back on adding `explicit`. I didn't expect this to cause such a widespread effect. Sorry. Now that you made all the necessary changes, we can keep it as-is. You decide if you prefer reverting these back. I'm good either way.

Reply:
I like current variant more. Just because it makes code more straightforward, I believe